consensus/src/extension/election.rs

```diff
@@ -63,11 +63,12 @@ impl<U: UnitWithParents> CandidateElection<U> {
 
     fn vote_from_parents(
         &mut self,
-        parents: &NodeMap<HashFor<U>>,
+        parents: Vec<HashFor<U>>,
+        parents_size: NodeCount,
```

Comment:
Perhaps it's clearer now to pass the threshold here, rather than the node count. Or at least rename this to `node_count`, due to the possible confusion I mentioned in another comment.